streaming/src/main/scala/org/apache/spark/streaming/scheduler/StreamingListenerBus.scala

```diff
@@ -51,12 +62,15 @@ private[spark] class StreamingListenerBus
     }
   }
 
-  override def onDropEvent(event: StreamingListenerEvent): Unit = {
-    if (logDroppedEvent.compareAndSet(false, true)) {
-      // Only log the following message once to avoid duplicated annoying logs.
-      logError("Dropping StreamingListenerEvent because no remaining room in event queue. " +
-        "This likely means one of the StreamingListeners is too slow and cannot keep up with the " +
-        "rate at which events are being started by the scheduler.")
-    }
+  /**
+   * Wrap StreamingListenerEvent as SparkListenerEvent so that it can be posted to Spark listener
```

Comment:
nit: Wrapper for StreamingListenerEvent as SparkListenerEvent...

Wraps --> verb, user for methods that does something. Class should be a noun.